### ISSUE TITLE
Xelatex issues with FontAwesome on macOS

### ISSUE THREAD

**Issue by benureau** [ReScience/ReScience-submission#32]
Compiling the article with the template provided by ReScience can lead to a confusing issue for users of macOS using MacTex, as I just experienced. It is well described here by someone else that stumbled into this one month ago: https://tex.stackexchange.com/questions/369068/how-can-i-use-the-fontawesome-package-with-xelatex

I solved this by opening the otf file located in `/usr/local/texlive/2017/texmf-dist/fonts/opentype/public/fontawesome/FontAwesome.otf` with Font Book, and installing it as a regular font.

We may want to include something about that in the `README.md` (possibly along with including `FontAwesome.otf` directly in the repo to make it easier) or solve the problem in some other way.

**Comment by ThomasA** [comment]
Yes, that would be me. I have not yet had time to find a solution to it and had to ask @rougier to recompile his paper himself so that I could publish it. I guess it *should* work fine on Linux as well, but for some reason it just doesn't on my system. I would be more than happy if we could come up with a "global" solution to the problem.

**Comment by rougier** [comment]
I think the font is only used for having the GitHub logo. Maybe we can use a regular pdf instead such that the font would not be necessary.

**Comment by mxochicale** [comment]
Just for the record, I experienced similar problems in GNU/Linux Ubuntu which creates this error:
```
kpathsea: Running mktexmf FontAwesome
! I can't find file `FontAwesome'.
```
Then, by following the recommendations at [:link:](https://tex.stackexchange.com/questions/132888/fontawesome-font-not-found), 
this line was added 
```
\defaultfontfeatures{Extension = .otf}
``` 
after `\usepackage{fontspec}` to `rescience-template.tex`.

Then the makefile works well!

**Comment by rougier** [comment]
Oh nice. Can you make a PR for the template ?

**Comment by ShanJiang21** [comment]
Another alternative is by using the package of  `awesome5 ` instead of awesome, and that could make a difference.

**Comment by rougier** [comment]
Thanks. We changed the template a few months ago and we do not use the awesome package anymore.
